### ISSUE TITLE
The model learns nothing after 3 epochs of training

### ISSUE THREAD

**Issue by geminiwenxu** [huggingface/transformers#9695]
I have trained a multilingual Bert model on 3 different input data configurations ( imbalanced, partial balanced, and full balanced) for the sentiment classification task. Everything works fine so far, except the zero-shot model being trainined on the full balanced dataset (training data: label balanced data; val/test data: label balanced data). however, the result is very weird:
<img width="638" alt="Screen Shot 2021-01-20 at 11 45 47 AM" src="https://user-images.githubusercontent.com/41744366/105165475-afb1b800-5b16-11eb-9d8f-d775fa9a07ee.png">

As you can see, the model has not learned anything, and it classifies everything into neutral in the testing phase.

Could anyone helps please?

**Comment by LysandreJik** [comment]
Hello, thanks for opening an issue! We try to keep the github issues for bugs/feature requests.
Could you ask your question on the [forum](https://discusss.huggingface.co) instead? You'll get more answers over there.

Thanks!
